app/views/mail_shared/_outlook_fix.html.erb

```diff
@@ -0,0 +1,1136 @@
+<style>
+img.no-float-image{
```

Comment:
@msecret Its overwriting a framework with existing styles in place for the img tag